frontend/src/components/recipe_card/recipe_card.js

```diff
@@ -35,11 +61,14 @@ const RecipeList = (props) => {
 }
 
 RecipeList.propTypes = {
-  text: PropTypes.string,
-  title: PropTypes.string,
   meal: PropTypes.string,
+  url: PropTypes.string,
   alternate: PropTypes.string,
-  name: PropTypes.string,
+  title: PropTypes.string,
+  snippet: PropTypes.string,
+  difficulty: PropTypes.string,
+  readyInMinutes: PropTypes.number,
+  spoons: PropTypes.number,
```

Comment:
I left a couple of comments above, but maybe we want to transform a few of these, like:
`url` -> `photo_url`
`readyInMinutes` -> `duration`
`spoons` -> `servings`

Reply:
Done!